app/process/bicycleRoadInfrastructureCenterline.lua

```diff
@@ -0,0 +1,141 @@
+package.path = package.path .. ";/app/process/helper/?.lua;/app/process/shared/?.lua"
+require("Set")
+require("FilterTags")
+require("ToNumber")
+-- require("PrintTable")
+require("AddAddress")
+require("MergeArray")
+require("AddMetadata")
+require("AddUrl")
+require("HighwayClasses")
+require("AddSkipInfoToHighways")
+require("StartsWith")
+
+
+local toTranslateTable = osm2pgsql.define_table({
+  name = 'bicycleRoadInfrastructureCenterline',
+  ids = { type = 'any', id_column = 'osm_id', type_column = 'osm_type' },
+  columns = {
+    { column = 'tags', type = 'jsonb' },
+    { column = 'geom', type = 'linestring' },
+    { column = 'offset', type='real'}
+  }
+})
+
+-- TODO: estimate the width based on the road type when `width=nil`
+--
+local function roadWidth(tags)
+  if tags["width"] ~= nil then
+    return tonumber(tags["width"])
+  end
+  if tags["est_width"] ~= nil then
+    return tonumber(tags["est_width"])
+  end
+  -- if tags["lanes"] ~= nil then
+  --   return tonumber(tags["lanes"]) * 2.5
+  -- end
+  local streetWidths = {primary=10, secondary=8, tertiary=6, residential=6}
+  if streetWidths[tags["highway"]] ~= nil then
+    return streetWidths[tags["highway"]]
+  end
+    -- if tags["highway"] == "cycleway" then
+  --   print(tags["cycleway"])
+  -- end
+  -- print(tags["highway"])
+  -- print(tags["lanes"])
+  -- print(tags["tracks"])
+  return 6
+end
+
+local function insert(object)
+  table:insert({
```

Comment:
Ich bin ziemlich sicher, dass hier der Name aus der Definition hin muss. Ich vermute, aktuell dürfte es gar nicht durchlaufen?

```suggestion
  toTranslateTable:insert({
```

Reply:
Oder anders, ich würde es weiterhin `table` nennen in Zeile 15, da jede Datei immer eine "table" hat.

Reply:
Habe ich umbenannt. die funktion wurde gar nicht mehr benutzt, deshalb hatte es trotzdem funktioniert